File: 07/forword_and_back.rb
require 'selenium-webdriver'

dr = Selenium::WebDriver.for :chrome
first_url = 'http://www.baidu.com'
puts "now access #{first_url}"
dr.get(first_url)
sleep 1
second_url = 'http://www.news.baidu.com'
puts "now access #{second_url}"
dr.get(second_url)
sleep 1

puts "back to #{first_url}"
dr.navigate.back()
sleep 1
puts "forward to #{second_url}"
dr.navigate.forward()
sleep 1
dr.quit()

File: 07/forword_and_back.py
from selenium import webdriver
from time import sleep
import os
if 'HTTP_PROXY'in os.environ: del os.environ['HTTP_PROXY']

dr = webdriver.Chrome()

first_url = 'http://www.baidu.com'
print "now access %s" %(first_url)

dr.get(first_url)
sleep(1)
second_url = 'http://www.news.baidu.com'
print "now access %s" %(second_url)
dr.get(second_url)
sleep(1)

print "back to %s" %(first_url)
dr.back()
sleep(1)
print "forward to %s" %(second_url)
dr.forward()
sleep(1)
dr.quit()
